pkgs/by-name/pa/passhole/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+}:
+let
+  pname = "passhole";
+  version = "1.10.0";
```

Comment:
I think these attributes make more sense at the top, since that would make it easier to maintain. If there is a version change for the package, it is much easier to just change it if it is at the top rather than hunt for it in the file.